mods/default/furnace.lua

```diff
@@ -166,6 +166,15 @@ local function furnace_node_timer(pos, elapsed)
 				else
 					-- Take fuel from fuel list
 					inv:set_stack("fuel", 1, afterfuel.items[1])
+					-- Put replacements in dst list or drop them on the furnace.
+					local replacements = fuel.replacements
+					if replacements[1] then
+						local leftover = inv:add_item("dst", replacements[1])
+						if not leftover:is_empty() then
+							local drop_pos = vector.new(pos.x, pos.y + 1, pos.z)
```

Comment:
Please search the closest air node in the area. Players stack their furnaces, thus most drops would be wasted.

Reply:
I agree.

Reply:
done